app/views/likes/_button.html.erb

```diff
@@ -1,7 +1,16 @@
+<style>
+  .lk{
+    border:none;
+    background-color: white;
+  }
+</style>
 <% like = current_user.likes.find_by(likeable: likeable) %>
```

Comment:
you should have used the query in controller or model, remove this

Reply:
okk i will write this in controller